Source/Core/Core/IOS/USB/Emulated/Skylander.cpp

```diff
@@ -19,465 +19,555 @@
+    {{3413, 0x4100}, {"Jet-Vac (Hurricane)", Game::Superchargers, Element::Air, Type::Skylander}},
+    {{3413, 0x4503},
+     {"Jet-Vac (Legendary Hurricane)", Game::Superchargers, Element::Air, Type::Skylander}},
+    {{3414, 0x4100},
+     {"Trigger Happy (Double Dare)", Game::Superchargers, Element::Tech, Type::Skylander}},
+    {{3414, 0x4502},
+     {"Trigger Happy (Power Blue)", Game::Superchargers, Element::Tech, Type::Skylander}},
+    {{3415, 0x4100},
+     {"Stealth Elf (Super Shot)", Game::Superchargers, Element::Life, Type::Skylander}},
+    {{3415, 0x4502},
+     {"Stealth Elf (Dark Super Shot)", Game::Superchargers, Element::Life, Type::Skylander}},
+    {{3415, 0x450F},
+     {"Stealth Elf (Instant)", Game::Superchargers, Element::Life, Type::Skylander}},
+    {{3416, 0x4100},
+     {"Terrafin (Shark Shooter)", Game::Superchargers, Element::Earth, Type::Skylander}},
+    {{3417, 0x4100},
+     {"Roller Brawl (Bone Bash)", Game::Superchargers, Element::Undead, Type::Skylander}},
+    {{3417, 0x4503},
+     {"Roller Brawl (Legendary Bone Bash)", Game::Superchargers, Element::Undead, Type::Skylander}},
+    {{3420, 0x4100}, {"Big Bubble Pop Fizz", Game::Superchargers, Element::Magic, Type::Skylander}},
+    {{3420, 0x450E},
+     {"Pop Fizz (Birthday Bash Big Bubble)", Game::Superchargers, Element::Magic, Type::Skylander}},
+    {{3421, 0x4100}, {"Eruptor (Lava Lance)", Game::Superchargers, Element::Fire, Type::Skylander}},
+    {{3422, 0x4100},
+     {"Gill Grunt (Deep Dive)", Game::Superchargers, Element::Water, Type::Skylander}},
+    {{3423, 0x4100},
+     {"Donkey Kong (Turbo Charge)", Game::Superchargers, Element::Life, Type::Skylander}},
+    {{3423, 0x4502},
+     {"Donkey Kong (Dark Turbo Charge)", Game::Superchargers, Element::Life, Type::Skylander}},
+    {{3424, 0x4100}, {"Bowser (Hammer Slam)", Game::Superchargers, Element::Fire, Type::Skylander}},
+    {{3424, 0x4502},
+     {"Bowser (Dark Hammer Slam)", Game::Superchargers, Element::Fire, Type::Skylander}},
+    {{3425, 0x4100}, {"Dive-Clops", Game::Superchargers, Element::Water, Type::Skylander}},
+    {{3425, 0x450E},
+     {"Dive-Clops (Missile-Tow)", Game::Superchargers, Element::Water, Type::Skylander}},
+    {{3425, 0x450F},
+     {"Dive-Clops (Instant)", Game::Superchargers, Element::Water, Type::Skylander}},
+    {{3426, 0x4100}, {"Astroblast", Game::Superchargers, Element::Tech, Type::Skylander}},
+    {{3426, 0x4503},
+     {"Astroblast (Legendary)", Game::Superchargers, Element::Light, Type::Skylander}},
+    {{3427, 0x4100}, {"Nightfall", Game::Superchargers, Element::Dark, Type::Skylander}},
+    {{3428, 0x4100}, {"Thrillipede", Game::Superchargers, Element::Life, Type::Skylander}},
+    {{3428, 0x450D},
+     {"Thrillipede (Eggcited)", Game::Superchargers, Element::Life, Type::Skylander}},
+    {{200, 0x2000}, {"Anvil Rain", Game::SpyrosAdv, Element::Other, Type::Item}},
```

Comment:
This is a very nitpicky comment - but previously we had these ordered by id, would be great if you could do the same and shift these up the list!

Reply:
I have split the list into types. If we were to sort everything by Id, magic items would get split apart from each other by game